internal/entitlement/postgresadapter/ent/schema/entitlement.go

```diff
@@ -35,8 +36,10 @@ func (Entitlement) Fields() []ent.Field {
 		field.Float("issue_after_reset").Optional().Nillable().Immutable(),
 		field.Bool("is_soft_limit").Optional().Nillable().Immutable(),
 		field.JSON("config", map[string]interface{}{}).Optional(),
-		field.Enum("usage_period_interval").Values(entitlement.UsagePeriodInterval("").StrValues()...).Optional().Nillable().Immutable(),
-		field.Time("usage_period_anchor").Optional().Nillable().Immutable(),
+		field.Enum("usage_period_interval").Values(recurrence.RecurrencePeriod("").Values()...).Optional().Nillable().Immutable(),
```

Comment:
in the db you use recurrence.RecurrencePeriod but in the types its entitlement.UsagePeriodInterval. lets make these consistent